README.md

```diff
@@ -1 +1,144 @@
-# To-Do-list
\ No newline at end of file
+<a name="readme-top"></a>
+
+  <h3><b>To Do LIst </b></h3>
+
+</div>
+
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+      - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 [Portfolio: Pop up window
+
+] <a name="about-project"></a>
+
+To do list is a project meant to master my skills in version control, GitHub flow, webpack and JavaScript.
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+- HTML & CSS
+- Linters
+- Git & Github
+- webpack
+- JavaScript
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+- **[Pop up window]**
+- **[contact form storage capability]**
+- **[addition of logo rotate on hover]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+## Getting Started
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- [Live Demo Link](https://kel-nana.github.io/To-Do-list/)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+**This is an example of how you may give instructions on setting up your project locally.**
+
+To get a local copy up and running follow these simple example steps.
+
+### Prerequisites
+
+- Git in your local machine
+- Code editor (VS code, Atom, etc)
+- Terminal
+
+### Setup
+
+- git clone git@github.com:Kel-nana/To-Do-list.git
+
+- cd To-Do-list
+
+### Usage
+
+- Add the changes you want and feel free to use in your project!
+
+### Run tests
+
+- Check that all the tests are correctly passed🤝
```

Comment:
- [ ] Everything is awesome, a small thing is missing on your README file to look like professional. I would suggest you, please provide how a user can install your project and start it on the browser like `npm install` and run on the browser `npm start`

Reply:
how a user can install your project and start it on the browser like [`npm install`](https://github.com/Kel-nana/To-Do-list/blob/e638e7f4dcff20cbc22456c398ebb6d5dd612d5f/README.md?plain=1#L78-L87) and run on the browser [`npm start`](https://github.com/Kel-nana/To-Do-list/blob/e638e7f4dcff20cbc22456c398ebb6d5dd612d5f/README.md?plain=1#L109-L112)

Reply:
@Kel-nana, Please write the command how to install and start the project on your README file.